cernopendata/modules/fixtures/data/docs/delphi-guide-eventserver/delphi-guide-eventserver.md

```diff
@@ -0,0 +1,36 @@
+# Raw data access: the DELPHI event server
+
+Raw data reconstruction is possible for the years 1992 and later, for individual events, via the DELPHI event server, `des`.
+
+## Requirements
+Running the even server requires access to the full DELPHI database, as well as the DELPHI raw data samples. It can be run from a container with access to the EOS file system, or locally from a machine with access to `cvmfs` and `EOS`.
```

Comment:
I would remove backquotes around CVMFS and EOS concepts. Better to have them as free words, and if we have corresponding terms in the glossary, people would get information what they are via mouse hover.

It's best to reserve backquotes only for things people can type as executables or file names or directory names etc.